server/src/test/java/org/opensearch/common/util/FeatureFlagTests.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.common.util;
+
+import org.opensearch.test.OpenSearchTestCase;
+
+public class FeatureFlagTests extends OpenSearchTestCase {
```

Comment:
I realize that these unit tests are rudimentary and don't actually test the `isEnabled == true` code path for `FeatureFlags`. I wasn't able to use `System.setProperty` - it gets flagged as a forbidden API. I would appreciate suggestions on how I could achieve better test coverage.

Reply:
I pushed a commit that grants write permissions on `opensearch.experimental.feature.*` so we can test the flags accordingly.